diff -rupN clean_ogre_cmake_project/CMakeLists.txt clean_ogre_cmake_project.new/CMakeLists.txt
--- clean_ogre_cmake_project/CMakeLists.txt	2013-08-19 19:50:11 +0100
+++ clean_ogre_cmake_project.new/CMakeLists.txt	2013-08-19 19:57:51 +0100
@@ -62,6 +62,12 @@ find_package(OIS REQUIRED)
 if(NOT OIS_FOUND)
 	message(SEND_ERROR "Failed to find OIS.")
 endif()
+
+# Find CEGUI 
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake_modules")
+find_package(CEGUI REQUIRED COMPONENTS OGRE)
+include_directories(CEGUI.h_DIR)
+set(OGRE_LIBRARIES ${OGRE_LIBRARIES} ${CEGUI_LIBRARY}) 
  
 # Find Boost
 if (NOT OGRE_BUILD_PLATFORM_IPHONE)
