diff -rupN clean_ogre_cmake_project/dist/bin/resources.cfg clean_ogre_cmake_project.new/dist/bin/resources.cfg
--- clean_ogre_cmake_project/dist/bin/resources.cfg	2013-08-19 21:34:17 +0100
+++ clean_ogre_cmake_project.new/dist/bin/resources.cfg	2013-08-19 21:32:27 +0100
@@ -6,5 +6,18 @@ Zip=../media/packs/SdkTrays.zip
 [General]
 FileSystem=../media
 FileSystem=../media/materials/scripts
+FileSystem=../media/materials/programs
 FileSystem=../media/materials/textures
 FileSystem=../media/models
+
+[Imagesets]
+FileSystem=path_to_cegui/imagesets
+[Fonts]
+FileSystem=path_to_cegui/fonts
+[Schemes]
+FileSystem=path_to_cegui/schemes
+[LookNFeel]
+FileSystem=path_to_cegui/looknfeel
+[Layouts]
+FileSystem=path_to_cegui/layouts
+
